wolfcrypt/test/test.c

```diff
@@ -6529,17 +6529,34 @@ static int EVP_test(const WOLFSSL_EVP_CIPHER* type, const byte* key,
         };
     #endif /* WOLFSSL_AES_192 */
 
-        Aes enc;
+#ifdef WOLFSSL_SMALL_STACK
+        Aes *enc = NULL;
+#else
+        Aes enc[1];
+#endif
         byte cipher[AES_BLOCK_SIZE * 4];
     #ifdef HAVE_AES_DECRYPT
-        Aes dec;
+#ifdef WOLFSSL_SMALL_STACK
+        Aes *dec = NULL;
+#else
+        Aes dec[1];
+#endif
         byte plain [AES_BLOCK_SIZE * 4];
     #endif
         int  ret = 0;
 
-	XMEMSET(&enc,0,sizeof enc);
+#ifdef WOLFSSL_SMALL_STACK
+    if ((enc = (Aes *)XMALLOC(sizeof *enc, NULL, DYNAMIC_TYPE_TMP_BUFFER)) == NULL)
```

Comment:
Please add `HEAP_HINT` to all XMALLOC/XFREE in test.c. Also for the `Aes` please use the new `DYNAMIC_TYPE_AES`. Thanks